src/components/AssetEventsPanel/interfaces.ts

```diff
@@ -73,4 +86,5 @@ export type AssetEventsPanelProps = WithAssetEventsProps &
 export type AssetEventsPanelPropsPure = WithAssetEventsDataProps &
   MetaEventsProps &
   AssetEventsPanelStylesProps &
-  AssetEventsPanelThemeProps;
+  AssetEventsPanelThemeProps &
+  RenderMetadataProps;
```

Comment:
`renderEventMetadata` has to be also present in `AssetEventsPanelProps` type, cause exactly this type will be exposed as a type described component props

It will also fix issue with prop table inside storybook:
https://pr-643.gearboxjs.preview.cogniteapp.com/?path=/docs/assets-asseteventspanel--basic-usage#available-props

Reply:
Added :)

Reply:
can't see commit with changes

Reply:
There kind of isn't one. I just smooshed it into the old one and force-pushed.

Reply:
ok, but I still can't see `renderEventMetadata` prop inside the prop table ) You can recheck yourself:
https://pr-643.gearboxjs.preview.cogniteapp.com/?path=/docs/assets-asseteventspanel--basic-usage#available-props

Reply:
It does seem to be very much missing. Let me take another look :)

Reply:
Well... mystery solved. I pushed the wrong remote. I am however having some troubles adding a comment to the prop as well. Is there perhaps some magic trick to it?

Reply:
Found the magic trick. Just had to restart Storybook and wait for 1min while my laptop-brick turned my legs into crisps.